distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -0,0 +1,359 @@
+
+[ditaa, catalog_architecture_data, png, ${image-width}]
+....
++------------------------------------------------------------+
+|                /-------------------\                       |
+|                |cDEFEndpoints      |                       |
+|                +------------+------+                       |
+|                |cDEF        |c369  |                       |
+|                | Operations | Data |                       |
+|/---------------+------------+------+------------+---------\|
+||cDEF           |cDEF               |cDEF        |cDEF     ||
+||  Transformers |                   | Federation | Sources ||
+|+---------------+ Catalog Framework +------------+---------+|
+||cDEF           |                   |cDEF   Eventing       ||
+||   Catalog     |                   +------------+---------+|
+||   Plugins     |                   |cDEF   Resources      ||
+|\---------------+-------------------+----------------------/|
+|                |cDEF               |                       |
+|                | Catalog Provider  |                       |
+|                +-------------------+                       |
+|                |cDEF               |                       |
+|                | Storage Provider  |                       |
+|                \-------------------/                       |
++------------------------------------------------------------+
+....
+
+The Catalog stores and translates Metadata, which can be transformed into many data formats, shared, and queried.
+The primary form of this metadata is the metacard. 
+A `Metacard` is a container for metadata. 
+`CatalogProviders` accept `Metacards` as input for ingest, and `Sources` search for metadata and return matching `Results` that include `Metacards`.
+
+=== Metacards
+
+A single instance of metadata in the Catalog (an instance of a metacard type) which generally contains metadata providing a title for the product and describing a product's geo-location, created and modified dates, owner or producer, security classification, etc. 
+
+==== Metacard Type
+
+A metacard type indicates the attributes available for a particular metacard.
+It is a model used to define the attributes of a metacard, much like a schema.
+
+===== Default Metacard Type and Attributes
+
+Most metacards within the system are created using with the default metacard type.
+The default metacard type of the system can be programmatically retrieved by calling `${ddf-branding}.catalog.data.BasicTypes.BASIC_METACARD`.
+The name of the default MetacardType can be retrieved from `${ddf-branding}.catalog.data.MetacardType.DEFAULT_METACARD_TYPE_NAME`.
+
+The default metacard type has the following required attributes.
+Though the following attributes are required on all metacard types, setting their values is optional except for ID.
+
+.[[_metacard_required_attributes]]Metacard Required Attributes
```

Comment:
Another strong candidate for programmatic generation, since this is the same information in the interfaces/impls.